lib/webapp.js

```diff
@@ -928,6 +929,27 @@ WebServiceHandle.prototype = {
         requestOptions.headers = headers;
       }
 
+      while(this.service && this.service.type === 'import' && serviceModified) {
+        let reqDataservices;
+        serviceModified = false;
+        const reqPlugin = pluginsArray.filter(plugin => plugin.identifier === this.service.sourcePlugin);
+        reqPlugin.filter(plugin => {
+          reqDataservices = plugin.dataServices.filter(dataService => dataService.name === this.service.sourceName);
```

Comment:
Wouldn't this produce a JS error if plugin.dataServices doesn't exist? As I understand it, dataServices are optional to plugins though I'm not all up to date with the context for this change